api/base/serializers.py

```diff
@@ -279,6 +299,53 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
+        instance_mapping = {item._id: item for item in instance}
```

Comment:
Okay, instead I am having the id_lookup be self.child.fields['id'].source.  Now everything is in the same place.